src/core/Expander/Expander.baseStyles.tsx

```diff
@@ -54,6 +58,7 @@ export const baseStyles = withSuomifiTheme(
     display: block;
     height: 0;
     overflow: hidden;
+    word-break: break-word;
```

Comment:
How is white-space defined for this component? Inherited from a reset etc?

Reply:
I think it comes from resets at the moment. It's set to normal, so any sequences of whitespaces are collapsed into one whitespace.